src/types/custom.ts

```diff
@@ -0,0 +1,20 @@
+import { Runtype, Rt, Static, create, validationError } from '../runtype'
+import { String } from './string'
+
+export interface Custom<A extends Rt, T extends string, K> extends Runtype<Static<A>> {
+  tag: T,
```

Comment:
>doesn't the args: K field already give you the power to stuff in whatever information you want?
Yes, you are right.
>tag should just be 'custom', because it's meant to be used when doing case analysis across the closed set of built-in runtypes
Hmm, i thought it was more properly. Why do you need a predefined and closed set of built-in runtypes? I'm not trying to argue. I'm just interested in your opinion :)